.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -91,17 +131,52 @@ steps:
       -e CLASSIC_BLOB=${STORAGE_ACCT_BLOB_URL} \
       -e CLASSIC_SAS_TOKEN="$(STORAGE_ACCT_SAS_TOKEN)" \
       -e OS_DISK_SAS=${OS_DISK_SAS} \
-      -e VHD_NAME=${VHD_NAME} \
```

Comment:
do we not need line 94?

Reply:
I don't think we need? az-copy uses `OS_DISK_SAS`. generate-sas uses `VHD_NAME`?